astroid/nodes/__init__.py

```diff
@@ -204,13 +204,89 @@
     YieldFrom,
 )
 
-# Can't create a proper __all__ with string because of a cyclic import for ClassDef
-__all__ = [
-    "CONST_CLS",
+__all__ = (
+    "AnnAssign",
+    "Arguments",
+    "Assert",
+    "Assign",
+    "AssignAttr",
+    "AssignName",
+    "AsyncFor",
+    "AsyncFunctionDef",
+    "AsyncWith",
+    "Attribute",
+    "AugAssign",
+    "Await",
+    "BinOp",
+    "BoolOp",
+    "Break",
     "builtin_lookup",
-    "are_exclusive",
```

Comment:
Not sure if we still need it in `pylint`

Reply:
We need it internally in astroid :D

Reply:
The comment was referring to the fact that you removed it from `__all__`. I haven't checked if it's used in `pylint`. If it is, it might make sense to keep it in `__all__`. Similar with `unpack_infer`.

Reply:
Ho my mistake, nice catch !